docs/how-payjoin-saves.md

```diff
@@ -0,0 +1,89 @@
+
+## Payjoin Payment Batching
+
+What if Dave the depositor can payjoin? He gets a benefit of breaking some privacy heuristics, and he can save the exchange some fees that might be able to be pased on to him. Payjoin lets the exchange fund withdrawals with Dave's deposit in the same transaction.
+
+```
+Dave     3 btc  ->  1 btc to Alice
+Exchange 2 btc      1 btc to Bob
+                    1 btc to Carol
+                   ~2 btc minus fees to Exchange
+```
+
+𝑏 + 2𝑖 + 4𝑜
+
+Since Dave's sweep to the exchange "[cut-through](./why-payjoin/scaling#transaction-cut-through)" to Alice, Bob, and Carol,
+
+1. The exchange never had to take on a new UTXO from Dave's deposit and pay fees to spend it.
+2. The exchange saved the fixed costs of making a second transaction, since Dave paid them.
+3. All parties enjoy better privacy since deposits and withdrawals are indistinguishable from an exchange consolidation, and these batched transactions are indistinguishable from individual transactions.
+
+Dave knows that Payjoin was used, but not which outputs are withdrawals vs consolidations.
+The Exchange can see everything as it could before, but an outside observer cannot without that information being leaked.
+Alice, Bob, and Carol can't tell whether or not all inputs came from the exchange or include an outsider's deposit. They know they got paid and that's their main concern.
+
+In total, the payjoin only cost
+
+*1 × base costs + 2 × input costs + 2 × output costs*
+
+saving at least 1/3 of the cost of a regular transaction, and fees were split between Dave and the Exchange by custom.[^1]
+
+## Future Payjoin Batching
+
+Bitcoin technically allows even more depositors to batch their transactions, too. Even though today's version of Payjoin only allows Dave to pay the exchange, future versions of Payjoin will allow others pay the exchange and each other in the same transaction too. More batching means more saving, and potentially more privacy too.
+
+```
+Dave     3 btc  ->  1 btc to Alice
+Erin     1 btc      1 btc to Bob
+Frank    2 btc      1 btc to Carol
+Exchange 2 btc     ~5 btc minus fees to Exchange
+```
+
+However, such transactions are more difficult to coordinate, so it will take an effort to develop this new protocol and get it deployed. Integrating Payjoin V2 can save money, improve privacy, and help get to the next iteration that massively fixes bitcoin's privacy through batching. Batching Bitcoin, saving sats, and preserving privacy seem like different goals, but with a little communication, the three goals can be achieved in every single transaction.
```

Comment:
```suggestion
However, such transactions are more difficult to coordinate, so it will take an effort to develop this new protocol and get it deployed. Integrating Payjoin V2 can save money, improve privacy, and help get to the next iteration that massively fixes Bitcoin's privacy through batching. Batching bitcoin, saving sats, and preserving privacy seem like different goals, but with a little communication, they can be achieved in every single transaction.
```